tsdb/store.go

```diff
@@ -237,7 +237,11 @@ func (s *Store) loadShards() error {
 				}
 
 				shard := NewShard(s.databaseIndexes[db], path, s.EngineOptions)
-				shard.Open()
+				err = shard.Open()
+				if err != nil {
+					s.Logger.Printf("failed to open shard %d: %s", shardID, err)
```

Comment:
Yeah, I'm not clear which is the correct way to go either.  It does feel like if a general `shard.Open()` fails something bad happened, and I would like to have more context on if we really should continue or not.